R-package/src/xgboost-win.def

```diff
@@ -0,0 +1,3 @@
+LIBRARY xgboost.dll
+EXPORTS
+ R_init_xgboost
```

Comment:
I don't think this is necessary, since the `R_init_xgboost` function already has the decoration `__declspec(dllexport)`.

Reply:
I think that decoration gets ignored when building it through the R toolchain.

Reply:
Good point! Forgot that only Visual Studio recognizes this tag.